pages/index.vue

```diff
@@ -2,10 +2,20 @@
   <div class="main">
     <h1 class="title">I Cannot Design.</h1>
     <hr>
+    <marquee direction="down" width="250" height="200" behavior="alternate">
+      <marquee behavior="alternate">
+        Under construction!!!1!1one
+      </marquee>
+    </marquee>
```

Comment:
Make sure to give the text here a background color so it is accessible. Might I suggest magenta?

We're about poor design choices, not poor accessibility. I want everyone to be able to enjoy this.

Reply:
Good call, let me add some additional things